recipes/shared-memory-dict/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set name = "shared-memory-dict" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/shared-memory-dict-{{ version }}.tar.gz
+  sha256: 8294257e1a2466afb5e19a878a8c446325b8913d57d6d9578ccba905564a974d
+
+build:
+  number: 0
+
+outputs:
+  - name: shared-memory-dict
+    build:
+      noarch: python
+    script: build_base.sh
+    requirements:
+      host:
+        - pip
+        - python >=3.6,<4.0
```

```suggestion
        - python >=3.8,<4.0
```
Upstream lists 3.8: https://github.com/luizalabs/shared-memory-dict/blob/ff8726973064afaec23eceb09b031abf535a8b4b/pyproject.toml#L21

---

Hmm, I guess `grayskull` missed that.